### PR TITLE
Los arreglitos que metimos

### PR DESCRIPTION
**Pull request by carvajal**
Summary:
Address the issue we found with old logging.
Fix some general structure things.

Test:
Manual on the web app
Logging and verifying

Rollback:
Yes. Feel free to rollback. Contact me if possible.

Screen Shot 2020-03-28 at 11 55 09 AM

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -33,18 +33,12 @@ class App extends React.Component {
...
-
+    bookInfo[target.id] = target.value;
     this.setState({ bookInfo });
   };
-  componentDidUpdate(pP, pS) {
-    console.log(this.state.bookInfo['Book Title']);
-    if (pS.bookInfo['Book Title'] !== this.state.bookInfo['Book Title']) {
-      this.setState({ bookinfo: this.state.bookInfo['Book Title'] });
-    }
-  }
```
Comment (carvajal) [inline src/components/App.js:47]: ah, I didn't realize we were not using this one

File: src/components/SearchForm.js [hunk 2]
```diff
@@ -32,15 +29,17 @@ class SearchForm extends React.Component {
       downloadFormat
     } = this.state;
 
+    const newSearchedTitle = this.props.bookInfo.bookTitle;
+
     //Calling the callback form parent component
     this.props.onFormSubmit({
-      searchedTitle,
+      searchedTitle: newSearchedTitle,
```
Comment (carvajal) [inline src/components/SearchForm.js:36]: nit: could we change the variable name so they all match? it is fine to do in a follow up

File: src/components/SearchForm.js [hunk 3]
```diff
@@ -49,9 +48,10 @@ class SearchForm extends React.Component {
       <Form onSubmit={this.onFormSubmit} className="externalForm">
         <Form.Group unstackable widths={2}>
           <Form.Input
+            id="bookTitle"
             label="Book Title"
             placeholder="Book Title"
-            value={this.props.bookInfo['Book Title']}
+            value={this.props.bookInfo['bookTitle']}
```
Comment (carvajal) [inline src/components/SearchForm.js]: could we use dot notation here too?

File: src/components/SearchForm.js [hunk 4]
```diff
@@ -91,17 +91,17 @@ class SearchForm extends React.Component {
             }}
           />
           <Form.Select
+            fluid
             label="Download Format"
-            control="select"
```
Comment (carvajal) [inline src/components/SearchForm.js:95]: why did you get rid of this one?

**Review by AdrianValdes — CHANGES_REQUESTED** [review]
Change everything you said.

**Comment by carvajal** [comment]
Done!

**Review by AdrianValdes — APPROVED** [review]
It looks cool!

**Pull request merged**